src/Symfony/Component/Form/CHANGELOG.md

```diff
@@ -11,7 +11,8 @@ CHANGELOG
    is deprecated. The method will be added to the interface in 6.0.
  * Implementing the `FormConfigBuilderInterface` without implementing the `setIsEmptyCallback()` method
    is deprecated. The method will be added to the interface in 6.0.
-
+ * Added `label_html` option to display the `label` text as HTML.
+ 
```

Comment:
extra space on this line :)

Reply:
done

Reply:
After that there is suddenly one error in AppVeyor, but I assume it has nothing to do with my changes.